### ISSUE TITLE
Mouse resets when i leave mousecape

### ISSUE THREAD

**Issue by steve-henderson** [alexzielenski/Mousecape#83]
When I move my cursor out of the mousecape application, it resets to the default cursor. How can I make it stay as the new cursor?

**Comment by alexzielenski** [comment]
This is very interesting. Does this still happen for you?

**Comment by steve-henderson** [comment]
Yes it still does. If I double click on the cursor it applies but the second I mouve it out of the application it resets to normal. And if I clicl apply cape the same thing happens.

**Comment by jukeboxbrass17** [comment]
@steve-henderson Same thing happens to me. I am running 10.11.6 El Capitan if that makes a difference.
